test/testcafe/spec/Identify_spec.js

```diff
@@ -246,3 +266,27 @@ test.requestHooks(identifyRequestLogger, identifyMockWithFingerprint)('should co
   const factorReqHeaders = factorReq.headers;
   await t.expect(factorReqHeaders['x-device-fingerprint']).notOk();
 });
+
+test.requestHooks(identifyRequestLogger, identifyMockWithFingerprintError)('should continue to compute device fingerprint if there are errors', async t => {
+  const identityPage = await setup(t);
+
+  await rerenderWidget({
+    features: {
+      deviceFingerprinting: true,
+    }
+  });
+
+  await identityPage.fillIdentifierField('Test Identifier');
+  await identityPage.clickNextButton();
+
```

Comment:
we should also check that `Next` button is now in `reloading` state once you click on it.. and then at line 290 this check `await t.expect(identityPage.getSaveButtonLabel()).eql('Next');` would make more sense..

Reply:
This is going to be problematic to test in `testcafe`, as the behavior of the test is to validate that we can proceed into the next step of the idx pipeline. We _could_ add `setTimeout` handlers to trigger the response after some set amount of time, but this a discouraged pattern.

IMO - this would be much better handled at the unit test layer, asserting the button changes state when pressed. Adding code coverage there, and additional `FormController` logic would be a great follow-up task!